packages/shared/routes/dashboard/settings/views/security/AppLock.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="typescript">
+    import { Dropdown, Text } from 'shared/components'
+    import { localize } from 'shared/lib/i18n'
+    import { activeProfile, updateProfile } from 'shared/lib/profile'
+
+    function assignTimeoutOptionLabel(timeInMinutes) {
```

Comment:
We need to decide on a convention for functions as we have a mixture of regular functions and arrow functions.

Reply:
used function only in this file